R/pkg/inst/tests/testthat/test_Windows.R

```diff
@@ -20,7 +20,7 @@ test_that("sparkJars tag in SparkContext", {
   if (.Platform$OS.type != "windows") {
     skip("This test is only for Windows, skipped")
   }
-  testOutput <- launchScript("ECHO", "a/b/c", capture = TRUE)
+  testOutput <- launchScript("ECHO", "a/b/c", wait = TRUE)
```

Comment:
from https://github.com/apache/spark/pull/16670#discussion_r97393450 this is "often times the result (error) is not enough to debug the issue."

our options are:
- redirect stdout always (could be very noisy)
- do not redirect stdout by default, but also do not drop - if R shell share the console with JVM (aka sparkR shell) then user would see messages, but if IDE, user would not see
  - I'd further propose an API to turn on redirect on demand which would address the R IDE case